content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -82,7 +82,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    If you do not have root access on the target system, you can still install kubectl to the `~/.local/bin` directory:
 
    ```bash
-   chmod +x kubectl
+   chmod +x ./kubectl
```

Comment:
Thanks for contributing to Kubernetes website. 😊

I'm wondering what's the reason for using `./file` instead of `file` as the argument to `chmod`?
Not sure which is better practice, hope you can provide some reference so that we can all learn from it.

Reply:
The original command is more idiomatic, but perhaps the replacement is better for teaching that `kubectl` is in the current working directory?